genesyscloud/resource_genesyscloud_user.go

```diff
@@ -124,84 +123,50 @@ func getAllUsers(ctx context.Context, sdkConfig *platformclientv2.Configuration)
 	// Newly created resources often aren't returned unless there's a delay
 	time.Sleep(5 * time.Second)
 
-	errorChan := make(chan error)
```

Comment:
Nice this eliminates complexity and frankly we are using this for a batch export process.